tests/full-version-label.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
```

Comment:
We should add `set -eo pipefail` to all three of these tests so the test is sure to fail on error.